src/FIOSDK.ts

```diff
@@ -601,7 +600,7 @@ export class FIOSDK {
       [{
         chain_code: chainCode,
         token_code: tokenCode,
-        public_address: publicAddress
+        public_address: publicAddress,
```

Comment:
please remove all of the changes that not relate to the feature, and create separate PR for it